shared/bull/create-redis.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import Redis from 'ioredis';
+Redis.Promise = require('bluebird');
```

Comment:
Node has built-in Promises now, why are we introducing bluebird, which is basically a Promise polyfill?

Reply:
Because the workers stopped running locally for me without doing this...no clue. Could you see if you can run athena locally with the latest `yarn` packages? Curious if you experience the same.

Reply:
It looks like the problem was us upgrading to ioredis 4.0. Related: https://github.com/OptimalBits/bull/pull/1156

I've downgraded us back to the previously-used version `3.2.2` for now.